playbooks/setup.yml

```diff
@@ -4,7 +4,9 @@
   become_user: '{{ become_user | default("root") }}'
   remote_user: '{{ remote_user | default("root") }}'
   gather_facts: '{{ gather_facts | default("yes") }}'
+  environment:
+    MINISHIFT_GITHUB_API_TOKEN: '{{ github_token | default("") }}'
   roles:
     - { role: prereqs, when: skip_prereqs|bool == false }
     - { role: minishift, when: setup_minishift|bool == true }
-    - role: pipeline
\ No newline at end of file
+    - role: pipeline
```

Comment:
This role don't exist in the new contra-env-setup because the goal of the new version is to use Helm.

Reply:
My editor just added new line at the end of file. I can try to remove it.

Reply:
the problem is the branch you are testing, this branch have some issues already fixed on PR #9

Reply:
Sure but I cannot see fix for this in #9.
We might wait with review after merging your PR. I just did not want to forget about this issue

Reply:
Totally agree @lslebodn thanks for share it.